modules/hcp-bm-add-nodes-to-inventory.adoc

```diff
@@ -0,0 +1,300 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-bm.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-bm-add-nodes-to-inventory_{context}"]
+= Creating an infrastructure environment and adding nodes
+
```

Comment:
I based this file on information that I found in the following lab exercise: https://labs.sysdeseng.com/hypershift-baremetal-lab/4.18/adding-bm-to-hw-inventory.html

If any of the information in this file should change, please let me know.